scripts/ClientSetup.ps1

```diff
@@ -111,6 +111,13 @@ $script = $webClient.DownloadString("https://${hostAddress}/repository/choco-ins
 choco config set cacheLocation $env:ChocolateyInstall\choco-cache
 choco config set commandExecutionTimeoutSeconds 14400
 
+if ($InternetEnabled) {
+    choco source add --name="'ChocolateyCore'" --source="'$RepositoryUrl'" --allow-self-service --admin-only --user="'$($Credential.UserName)'" --password="'$($Credential.GetNetworkCredential().Password)'" --priority=0
+}
+else {
+    choco source add --name="'ChocolateyCore'" --source="'$RepositoryUrl'" --allow-self-service --admin-only --priority=0
+}
+
 if ($InternetEnabled) {
     choco source add --name="'ChocolateyInternal'" --source="'$RepositoryUrl'" --allow-self-service --user="'$($Credential.UserName)'" --password="'$($Credential.GetNetworkCredential().Password)'" --priority=1
```

Comment:
So, this is adding the second source with the same URL as the block above with `ChocolateyCore` unless I'm mistaken?

Do we need to pass in both URLs?

Reply:
Also, we were turning off self-service/making the source admin only for the Internal feed, right?

Reply:
Sorted with Regex